src/open_inwoner/utils/css.py

```diff
@@ -0,0 +1,64 @@
+from typing import List
+
+import tinycss2
+from bleach.css_sanitizer import (
+    ALLOWED_CSS_PROPERTIES as _ALLOWED_CSS_PROPERTIES,
+    ALLOWED_SVG_PROPERTIES as _ALLOWED_SVG_PROPERTIES,
+)
+
+# alias for clarity / override
+ALLOWED_CSS_PROPERTIES = _ALLOWED_CSS_PROPERTIES
+ALLOWED_SVG_PROPERTIES = _ALLOWED_SVG_PROPERTIES
+
+
+def allowed_properties() -> List[str]:
+    # used for admin etc
+    parts = set()
+    parts.update(ALLOWED_CSS_PROPERTIES)
+    parts.update(ALLOWED_SVG_PROPERTIES)
+    return list(sorted(parts))
```

Comment:
The properties from bleach come in the form of a `frozenset`, which is assigned to `ALLOWED_CSS_PROPERTIES` and `ALLOWED_SVG_PROPERTIES` above. In this function you first build a `set` and then return a `list` Why?

If we don't intend to mutate these structures, it would be better to stick to `frozenset`, which is immutable.

Reply:
I'm going to strip this list and remove some of the weird properties nobody would use.